src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,27 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+const getLinkClass = (
+  { isActive }: { isActive: boolean },
+) => classNames(
+  'navbar-item',
+  { 'navbar-item has-background-grey-lighter': isActive },
+);
```

Comment:
Why set the `navbar-item` class twice?
```suggestion
) => classNames(
  'navbar-item',
  { 'has-background-grey-lighter': isActive },
);
```